### PR TITLE
feat: Add 'submit --from' to CronWorkflow and WorkflowTemplate in UI. Closes #3112

### LINKED ISSUE #3112: Cron Workflow Run Now
# Summary

Allow Cron Workflow to run at any arbitrary time with a button trigger

# Motivation

Even though the Cron is suppose to run on a schedule. The Kubernete dashboard's cronjob supports a way to trigger the job as needed. 

# Proposal

Add a date picker and run button to the cron tab so a default `DS` value could be passed into the cron workflow to enable run on the given workflow at any arbitrary time.

### PR DESCRIPTION
**Pull request by simster7**
Closes: #3112
Fixes: #3092

### CODE REVIEW

File: ui/src/app/cron-workflows/components/cron-workflow-details/cron-workflow-details.tsx [hunk 1]
```diff
@@ -61,10 +61,15 @@ export class CronWorkflowDetails extends BasePage<RouteComponentProps<any>, Stat
...
+                            {
+                                title: 'Submit Now',
+                                iconClassName: 'fa fa-plus',
+                                action: () => this.submitCronWorkflow()
+                            },
                             {
                                 title: 'Delete',
                                 iconClassName: 'fa fa-trash',
-                                action: () => this.deleteWorkflowTemplate()
+                                action: () => this.deleteCronWorkflow()
```
Comment (alexec) [inline ui/src/app/cron-workflows/components/cron-workflow-details/cron-workflow-details.tsx:72]: ek!

File: ui/src/app/cron-workflows/components/cron-workflow-details/cron-workflow-details.tsx [hunk 2]
```diff
@@ -93,7 +98,21 @@ export class CronWorkflowDetails extends BasePage<RouteComponentProps<any>, Stat
         );
     }
 
-    private deleteWorkflowTemplate() {
+    private submitCronWorkflow() {
+        services.workflows
+            .submit('cronwf', this.name, this.namespace)
+            .catch(e => {
```
Comment (alexec) [inline ui/src/app/cron-workflows/components/cron-workflow-details/cron-workflow-details.tsx:104]: catch after then?

Comment (simster7) [reply ui/src/app/cron-workflows/components/cron-workflow-details/cron-workflow-details.tsx:104]: Done!

**Comment by alexec** [comment]
Does this also fix #3092 ?

**Comment by simster7** [comment]
> Does this also fix #3092 ?

Yessir

**Pull request merged**